### DISCUSSION TITLE
Is the smtp server optional or required?

### DISCUSSION THREAD

**Discussion by soupdiver** [plausible/analytics#979] [Self-Hosted Support] [answered]
I just need a simple minimal solution to gather some stats for a small site.
That's why I wondered a little bit about the smtp-server portion in the deployment.
Can I simply omit the smtp server or will this cause trouble down the road?

**Comment by ukutaht** [comment, marked as answer]
I'm running an instance with no email stuff configured at all. It works OK but obviously you lose out on email reports and stuff like that.

You will need to activate your account manually as explained here: https://github.com/plausible/analytics/discussions/700#discussioncomment-356212
